### PR TITLE
Add petpet command

### LINKED ISSUE #1197: PetPet command
`s!petpet`

8259f921-17ea-4a6a-8221-341685dabdf8

---

image

https://www.npmjs.com/package/gif-encoder

https://benisland.neocities.org/petpet/

### PR DESCRIPTION
**Pull request by blobfysh**
Adds petpet command for petting your friends, closes #1197 

patpatpatpatpatpatpatpatpatpat

### CODE REVIEW

File: src/commands/image-manipulation/petpet.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+module.exports = class Petpet extends Command {
+  constructor (client) {
+    super({
+      name: 'petpet',
+      aliases: ['patpat'],
+      category: 'images',
+      requirements: { canvasOnly: true },
+      parameters: [{
+        type: 'image',
+        missingError: 'commands:morejpeg.missingImage'
```
Comment (pedrofracassi) [inline src/commands/image-manipulation/petpet.js:14]: Create an error string for this command instead of uring the one from morejpeg

**Review by pedrofracassi — CHANGES_REQUESTED** [review]

File: src/commands/image-manipulation/petpet.js (hunk 1 above)
Comment (blobfysh) [reply src/commands/image-manipulation/petpet.js:14]: will do

**Comment by blobfysh** [comment]
fixed

**Review by pedrofracassi — APPROVED** [review]
image

**Review by Doges — APPROVED** [review]
LGTM

**Pull request merged**